modules/eco-checking_status_of_node_maintenance_cr_tasks.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-maintenance-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-checking_status_of_node_maintenance_cr_tasks_{context}"]
+= Checking status of current NodeMaintenance CR tasks
+
+You can check the status of current `NodeMaintenance` CR tasks.
+
+.Prerequisites
+
+* Install the {product-title} CLI `oc`.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+* Check the status of current node maintenance tasks by running the following command:
++
+[source,terminal]
+----
+$ oc get nm -o yaml
+----
++
+.Example output
++
+[source,yaml]
+----
+apiVersion: v1
+items:
+- apiVersion: nodemaintenance.medik8s.io/v1beta1
+  kind: NodeMaintenance
+  metadata:
+...
+  spec:
+    nodeName: node-1.example.com
+    reason: Node maintenance
+  status:
+    evictionPods: 3   <1>
+    phase: Succeeded
+    lastError: "Last failure message" <2>
+    totalpods: 5
+...
+----
+<1> `evictionPods` is the number of pods scheduled for eviction.
+<2> `lastError` records the latest eviction error, if any.
```

Comment:
Just for consistency, since the parameter names are not included in the annotations above.
```suggestion
<1> The number of pods scheduled for eviction.
<2> The latest eviction error, if any.
```